RecoLocalCalo/HGCalRecProducers/plugins/HGCalUncalibRecHitWorkerWeights.cc

```diff
@@ -73,16 +76,10 @@ HGCalUncalibRecHitWorkerWeights::HGCalUncalibRecHitWorkerWeights(const edm::Para
 }
 
 void HGCalUncalibRecHitWorkerWeights::set(const edm::EventSetup& es) {
-  if (uncalibMaker_ee_.isSiFESim()) {
-    uncalibMaker_ee_.setGeometry(&es.getData(ee_geometry_token_));
-  }
-  if (uncalibMaker_hef_.isSiFESim()) {
-    uncalibMaker_hef_.setGeometry(&es.getData(hef_geometry_token_));
-  }
-  uncalibMaker_heb_.setGeometry(nullptr);
-  if (uncalibMaker_hfnose_.isSiFESim()) {
-    uncalibMaker_hfnose_.setGeometry(&es.getData(hfnose_geometry_token_));
-  }
+  uncalibMaker_ee_.setGeometry(es.getHandle(ee_geometry_token_), uncalibMaker_ee_.isSiFESim());
```

Comment:
What is the meaning of passing the result of the `isSiFESim()` method to the `setGeometry()` method of the very same class?

Reply:
can't we just use directly the value of `isSiFESim_` inside the `setGeometry()` method and be sure that that value has the correct meaning for all the use cases?

Reply:
sure, I will update the configuration file [1], setting the flag isSiFESim() for HGCHEB to false and use directly the isSiFESim_ method in setGeometry.

[1] https://github.com/cms-sw/cmssw/blob/809ee86e5b4e861766f27724f788d9f8724aae69/RecoLocalCalo/HGCalRecProducers/python/HGCalUncalibRecHit_cfi.py